snabbdom.js

```diff
@@ -45,10 +46,13 @@ function init(modules) {
   }
 
   function createElm(vnode, insertedVnodeQueue) {
-    var i, data = vnode.data;
+    var i, thunk, data = vnode.data;
```

Comment:
Is it just a vocabulary issue? Since we descend into vnode.data.vnode, I needed to keep a reference on the starting point of the descent. Since it's only used for thunk or thunk-like things, I called the variable "thunk", but we could probably find a better name. But we can't use "parent" as it would be ambiguous. Do you have a suggestion?